modules/rhcos-enabling-multipath.adoc

```diff
@@ -1,119 +1,34 @@
-
-+
-[source,yaml]
-----
-apiVersion: machineconfiguration.openshift.io/v1
-kind: MachineConfig
-metadata:
-  labels:
-    machineconfiguration.openshift.io/role: "master"
-  name: 99-master-kargs-mpath
-spec:
-  kernelArguments:
-    - rd.multipath=default
-    - root=/dev/disk/by-label/dm-mpath-root
-----
-
-. To enable multipathing on worker nodes:
-
-* Create a machine config file, such as `99-worker-kargs-mpath.yaml`, that instructs the cluster to add the `worker` label and that identifies the multipath kernel argument, for example:
-+
-[source,yaml]
-----
-apiVersion: machineconfiguration.openshift.io/v1
-kind: MachineConfig
-metadata:
-  labels:
-    machineconfiguration.openshift.io/role: "worker"
-  name: 99-worker-kargs-mpath
-spec:
-  kernelArguments:
-    - rd.multipath=default
-    - root=/dev/disk/by-label/dm-mpath-root
-----
-
-. Create the new machine config by using either the master or worker YAML file you previously created:
-+
-[source,terminal]
-----
-$ oc create -f ./99-master-kargs-mpath.yaml
-----
-
-. Check the machine configs to see that the new one was added:
-+
-[source,terminal]
-----
-$ oc get MachineConfig
-----
-+
-.Example output
-[source,terminal]
-----
-NAME                                               GENERATEDBYCONTROLLER                      IGNITIONVERSION   AGE
-00-master                                          52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-00-worker                                          52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-01-master-container-runtime                        52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-01-master-kubelet                                  52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-01-worker-container-runtime                        52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-01-worker-kubelet                                  52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-99-master-kargs-mpath                              52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             105s
-99-master-generated-registries                     52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-99-master-ssh                                                                                 3.2.0             40m
-99-worker-generated-registries                     52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-99-worker-ssh                                                                                 3.2.0             40m
-rendered-master-23e785de7587df95a4b517e0647e5ab7   52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-rendered-worker-5d596d9293ca3ea80c896a1191735bb1   52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-----
-
-. Check the nodes:
+. To enable multipathing at installation time, append the kernel arguments by entering the following command:
```

Comment:
There's two facets actually for a fully multipathed installation:
1. We want `coreos-installer install` to write to the multipathed device. This means that users should use e.g. `coreos-installer install /dev/mapper/mpatha` or using a WWN-named link under `/dev/disk/by-id`. That requires turning on multipath in the installation environment, which could be done using `rd.multipath=default` in the PXE/ISO boot, or just manually calling e.g. `mpathconf --enable` and `systemctl start multipathd.service` before calling `coreos-installer install`.
2. Appending kargs to the `coreos-installer install` command so that the installed system itself will use multipath right from the first boot. This part you already have.

Reply:
OK, thanks for the details. I've added an initial step based on whether you are doing an ISO/PXE boot or not. But I'm not sure if I should incorporate this part of your comment: 
>  This means that users should use e.g. `coreos-installer install /dev/mapper/mpatha` or using a WWN-named link under `/dev/disk/by-id`.

If so, how would you recommend doing so? As a prerequisite like the following perhaps?

```
.Prerequisites
* You are using either `coreos-installer install /dev/mapper/mpatha` or a World Wide Name (WWN) link under `/dev/disk/by-id`.
```